cookbook/deployment/fortrabbit.rst

```diff
@@ -0,0 +1,230 @@
+.. index::
+   single: Deployment; Deploying to fortrabbit.com
+
+Deploying to fortrabbit
+=======================
+
+This step-by-step cookbook describes how to deploy a Symfony web application to
+ `fortrabbit`_. You can read more about using Symfony with fortrabbit on the 
+ official fortrabbit `Symfony install guide`_.
+
+
+
+Setting up fortrabbit
+---------------------
+
+Before we begin, you should have done a few things on the fortrabbit side:
+
+- Sign up
+- Add an SSH key to your Account (to deploy via Git)
+- Create an App
+
+
+Preparing your Application
+--------------------------
+
+You don't need to change any code to deploy a Symfony application to fortrabbit. 
+But it requires some minor tweaks to its configuration.
+
+
+Configure Logging
+~~~~~~~~~~~~~~~~~
+
+Per default Symfony logs to a file. Modify the app/config/config_prod.yml file 
+to redirect it to PHP's error_log():
+
+.. code-block:: yaml
+
+   monolog:
+   # ..
+   handlers:
```

Comment:
Indentation is wrong here and you could also add XML and PHP configuration examples:

```rst
.. configuration-block::

    .. code-block:: yaml

        # app/config/config_prod.yml
        monolog:
            # ...
            handlers:
                nested:
                    type: error_log

    .. code-block:: xml

        <!-- app/config/config_prod.xml -->
        <?xml version="1.0" encoding="UTF-8" ?>
        <container xmlns="http://symfony.com/schema/dic/services"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xmlns:monolog="http://symfony.com/schema/dic/monolog"
            xsi:schemaLocation="http://symfony.com/schema/dic/services
                http://symfony.com/schema/dic/services/services-1.0.xsd
                http://symfony.com/schema/dic/monolog
                http://symfony.com/schema/dic/monolog/monolog-1.0.xsd">

            <monolog:config>
                <!-- ... -->
                <monolog:handler
                    name="nested"
                    type="error_log"
                />
            </monolog:config>
        </container>

    .. code-block:: php

        // app/config/config_prod.php
        $container->loadFromExtension('monolog', array(
            // ...
            'handlers' => array(
                'nested' => array(
                    'type' => 'error_log',
                ),
            ),
        ));
```